client_test.go

```diff
@@ -151,11 +151,21 @@ func TestNewWithMultipleNodes(t *testing.T) {
 
 	nns := conf.Namenodes()
 
-	nns = append([]string{"localhost:100"}, nns...)
+	nns = append([]string{removeEmptyStrings(nns)[0]}, nns...)
```

Comment:
if it is empty, there will be index out of bounds exception

Reply:
It is never empty